3rdparty/CMakeLists.txt

```diff
@@ -11,11 +11,6 @@ set(FIBER_SHARED OFF)
 set(FIBER_OBJECT ON)
 add_subdirectory(fiber)
 
-set(BENCHMARK_ENABLE_GTEST_TESTS OFF)
-set(BENCHMARK_ENABLE_TESTING OFF)
-set(BENCHMARK_ENABLE_INSTALL OFF)
-add_subdirectory(benchmark)
-
```

Comment:
I assume this is because the fiber benchmark is being removed, not sure if that's on purpose  (see question below)